### DISCUSSION TITLE
Select segments by label

### DISCUSSION THREAD

**Discussion by robinmattern** [mifi/lossless-cut#1308] [Q&A]
Am I missing how this works, or is it an issue? 
![image](https://user-images.githubusercontent.com/1158299/189456918-c2fde139-e6fc-4cc2-8170-a1a5bd7d52d7.png)

**Comment by mifi** [comment]
Hi. Currently the label has to be an exact match. I added your suggestions here: #427

**Reply by mifi** [reply]
in the latest version it's possible to select by JS expression.
